packages/syntax-core/src/RadioButton/RadioButton.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import { StoryObj, Meta } from "@storybook/react";
+import RadioButton from "./RadioButton";
+import React, { useState } from "react";
+
+export default {
+  title: "RadioButton",
+  component: RadioButton,
+  parameters: {
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/p7LKna9JMU0JEkcKamzs53/%F0%9F%93%90-Syntax?node-id=1007-4104&t=Vt5ql6LLs6d29szu-0",
+    },
+  },
+  argTypes: {
+    checked: {
+      control: "boolean",
+    },
+    size: {
+      options: ["sm", "md"],
+      control: { type: "radio" },
+    },
+    disabled: {
+      control: "boolean",
+    },
+    error: { control: "boolean" },
+    label: { control: "string" },
```

Comment:
@JFChoe should this be `control: "text"`? I'm currently not able to update it in Storybook

![image](https://user-images.githubusercontent.com/127199/229622154-5d032cd1-241f-4a4c-83fd-b7bed63b95cd.png)